### PR TITLE
End screen

### PR DESCRIPTION
**Pull request by noctera**
| Status  | Type  |
| :---: | :---: |
| ✅ Ready | Feature |

## TODO

- [x] do not mutate state directly
- [x] move card to own component
- [x] learn -> no packages found screen
- [x] ~~fix direction screen flags due to language relation~~ -> fixed by #62 
- [x] round percentage 
image
- [x] use translations
- [x] remove || - in end screen table due to falsey values 
- [x] Clear query in fresh learning start
- [x] Move congratulations into own component 
- [x] Improve congratulations translations 

## Description

Added end screen when query is finished

## Screenshots / GIFs (if appropriate):

will add later

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

File: src/Components/PackageOverview/PackageOverview.jsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { useHistory } from "react-router-dom";
 
 import Button from "../Button/Button.jsx";
 
-import "../../redux/Actions/learn.js";
-import { setLearnedPackage } from "../../redux/Actions/learn.js";
+import "../../redux/Actions/query.js";
```
Comment (luwol03) [inline src/Components/PackageOverview/PackageOverview.jsx:8]: this import make no sense.

File: src/screens/Learn/End/End.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import Table from "../../../Components/Table/Table.jsx";
+
+import { clearQuery } from "../../../redux/Actions/query.js";
+
+import "./End.scss";
+
+const End = () => {
+  const correctVocabs = useSelector((state) => state.query.correct);
+  const wrongVocabs = useSelector((state) => state.query.wrong);
+  const [percentage] = useState(
```
Comment (luwol03) [inline src/screens/Learn/End/End.jsx:16]: why is this a state?

**Review by luwol03 — CHANGES_REQUESTED** [review]

File: src/screens/Learn/End/End.jsx (hunk 2 above)
Comment (noctera) [reply src/screens/Learn/End/End.jsx:17]: is this resolved? I could change it to a normal const variable

File: src/Components/Congratulation/Congratulation.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import "./Congratulation.scss";
+
+const Congratulation = (percentage) => {
+  const { t } = useTranslation();
+  const [congratulation, setCongratulations] = useState();
+
+  useEffect(() => {
+    //get size of congratulation array and calculate the message with the given percentage
+    const translations = t("screens.endScreen.congratulations", {
```
[the remark this answers is not in the document]
Comment (noctera) [reply src/Components/Congratulation/Congratulation.jsx]: done

**Review by luwol03 — DISMISSED** [review]

**Review by luwol03 — APPROVED** [review]

**Review by zikowang — APPROVED** [review]

**Pull request merged**